packages/graphql/src/schemaTypes/objectTypes/gql-Migration.ts

```diff
@@ -14,7 +14,7 @@ export const Migration = objectType({
     t.nonNull.field('step', {
       type: MigrationStepEnum,
       description: 'Step where the migration is right now',
-      resolve: () => 'renameManual',
+      resolve: () => 'configFile',
```

Comment:
it sounds odd that the config would be blocked to a step. 
Maybe wiring has been missed?

Reply:
I figured this was temporary as we talked about switching this to client side. Right now I just had it so I could actually see what was going on.